src/client/app/containers/app.ts

```diff
@@ -23,7 +24,7 @@ import { isNil } from 'lodash';
                     <command icon="PageCheckedin" title="${Strings.shareMenuPublic}" (click)="shareGist({isPublic: true, isUpdate: false})"></command>
                     <command icon="ProtectedDocument" title="${Strings.shareMenuPrivate}" (click)="shareGist({isPublic: false, isUpdate: false})"></command>
                     <command id="CopyToClipboard" icon="Copy" title="${Strings.shareMenuClipboard}" (click)="shareCopy()"></command>
-                    <command icon="Download" title="${Strings.shareMenuExport}" (click)="shareExport()"></command>
+                    <command [hidden]="!isDownloadZipSupported" icon="Download" title="${Strings.shareMenuExport}" (click)="shareExport()"></command>
```

Comment:
Agreed, makes sense. Or you can grey it out with a title saying this feature is not supported.